src/scriptdev2/scripts/eastern_kingdoms/naxxramas/boss_razuvious.cpp

```diff
@@ -161,3 +159,5 @@ void AddSC_boss_razuvious()
     pNewScript->GetAI = &GetAI_boss_razuvious;
     pNewScript->RegisterSelf();
 }
+Contact GitHub API Training Shop Blog About
```

Comment:
dont know where those line come from but its pretty obvious its wrong and cause build fail